openpower/package/hcode/hcode.mk

```diff
@@ -0,0 +1,50 @@
+################################################################################
+#
+# HCODE
+#
+################################################################################
+
+HCODE_VERSION ?= 270068cbfcb001fa5ccfbb6e8bc18eea22d68487
+HCODE_SITE = $(call github,open-power,hcode,$(HCODE_VERSION))
+
+HCODE_LICENSE = Apache-2.0
+
+HCODE_INSTALL_IMAGES = YES
+HCODE_INSTALL_TARGET = NO
+
+HCODE_DEPENDENCIES = host-binutils host-ppe42-gcc hostboot-binaries
+
+HW_IMAGE_BIN_PATH=output/images/hw_image
+HW_IMAGE_BIN=p9n.hw_image.bin
+HCODE_IMAGE_BIN = p9n.ref_image.bin
+
+CROSS_COMPILER_PATH=$(PPE42_GCC_BIN)
+PPE_TOOL_PATH ?= $(CROSS_COMPILER_PATH)
+PPE_PREFIX    ?= $(PPE_TOOL_PATH)/bin/powerpc-eabi-
+
+ifeq ($(wildcard /opt/rh/devtoolset-*),)
+CXXPATH= $(dir $(HOSTCXX))
```

Comment:
We do we have to explicitly set the CXXPATH in all cases? Isn't it set by default?

Reply:
could be a bad choice of a name I don't see it being used for anything else though, the intent was to grab the path for the local compiler and use it for __EKB_PATH, I think HOSTCXX gets set for all cases by buildroot - but I needed to be able to point to the compiler in the deftoolset dir on pool systems

Reply:
This won't go upstream, IBM should fix their systems to not be ancient and broken.

i.e. with what I just merged upstream, you're going to have to manually set PATH on those ancient pool machines. I'll ping folk on internal Slack too.